developer-docs-site/static/api/devnet-api.html

```diff
@@ -11,7 +11,7 @@
   </head>
   <body>
     <elements-api id="spec_viewer"
-      apiDescriptionUrl="https://raw.githubusercontent.com/aptos-labs/aptos-core/devnet/api/doc/openapi.yaml"
+      apiDescriptionUrl="https://raw.githubusercontent.com/aptos-labs/aptos-core/93ce59d9f9fb7af99c47f0384d7967e3ae265c81/api/doc/v0/openapi.yaml"
```

Comment:
I think it should probably be this: https://raw.githubusercontent.com/aptos-labs/aptos-core/devnet/api/doc/v1/spec.yaml. We want to point to the tip of the devnet branch and use the v1 spec.

Reply:
It was v0 before. So is the v0 completely unsupported now? (as in we are not linking to it?). The index page says "Latest" and "Devnet", which both point to v1 but on main and devnet, correct?

Reply:
That's right, we're gonna delete v0 from the code in a few days and from the days in 8 days.